src/main/java/org/zalando/nakadi/controller/EventPublishingController.java

```diff
@@ -69,37 +75,52 @@ public ResponseEntity postEvent(@PathVariable final String eventTypeName,
             throws AccessDeniedException, BlockedException, ServiceTemporarilyUnavailableException,
             InternalNakadiException, EventTypeTimeoutException, NoSuchEventTypeException {
         LOG.trace("Received event {} for event type {}", eventsAsString, eventTypeName);
-
+        final HttpServletRequest httpServletRequest = request.getNativeRequest(HttpServletRequest.class);
+        final Scope publishingScope = TracingService
```

Comment:
Is it possible that a `span` fails to be instantiated?  I'm just asking to make sure that broken instrumentation would not cause Nakadi to fail.

Reply:
I did not see any exception being declared in the opentracing definitions. AFAIK it suppresses failures.